### DISCUSSION TITLE
Restoring older dependencies from gradle cache

### DISCUSSION THREAD

**Discussion by MilanVadhel** [actions/cache#732] [Q&A]
Hello
I have used GitHub action cache in one of my private repository and it works well and reduces workflow running time. But I have one question is that I have added some required dependencies in my build.gradle file in the android project and what if I update any dependency version in the build.gradle file? Will it take an older dependency from the cache or create a new cache with a new dependency?

Thanks

**Comment by jongwooo** [comment]
@MilanVadhel Please refer to the following links.
[https://github.com/actions/cache/blob/main/examples.md#java---gradle](https://github.com/actions/cache/blob/main/examples.md#java---gradle)
[https://aboutfilipe.com/posts/faster-android-builds-using-github-actions/](https://aboutfilipe.com/posts/faster-android-builds-using-github-actions/)
